src/librustc/middle/infer/combine.rs

```diff
@@ -354,50 +356,55 @@ pub trait Combineable<'tcx> : Repr<'tcx> + TypeFoldable<'tcx> {
 impl<'tcx,T> Combineable<'tcx> for Rc<T>
     where T : Combineable<'tcx>
 {
-    fn combine<C:Combine<'tcx>>(combiner: &C,
-                                a: &Rc<T>,
-                                b: &Rc<T>)
-                                -> cres<'tcx, Rc<T>>
+    fn combine<C:Combine<'tcx>>(
+               combiner: &C,
+               a: &Rc<T>,
+               b: &Rc<T>)
+               -> cres<'tcx, Rc<T>>
```

Comment:
I don't think we have a precedent for this elsewhere, but if you find the rightward-drift a bit annoying you can always throw the `C: Cmbine<'tcx>` into a where clause to shorten the generics list. I think it's more common for the arguments to be aligned against the `(` than not.